sdk/python/tests/integration/registration/test_universal_types.py

```diff
@@ -230,7 +230,7 @@ def test_feature_get_online_features_types_match(online_types_test_fixtures):
 
     driver_id_value = "1" if config.entity_type == ValueType.STRING else 1
     online_features = fs.get_online_features(
-        features=features, entity_rows=[{"driver": driver_id_value}],
+        features=features, entity_rows=[{"driver_id": driver_id_value}],
```

Comment:
since you're fixing this right now, you might as well also fix this in `test_s3_custom_endpoint.py` (line 48)